### PR TITLE
Add note about the supportsReadOnly mode config

### PR DESCRIPTION
**Pull request by sofietoft**
This should not be merged before I've cleared up the difference between the `supportsReadOnly` setting and the `isReadOnly` setting.

### CODE REVIEW

File: Extending/Property-Editors/Package-Manifest/index.md [hunk 1]
```diff
@@ -111,15 +113,17 @@ The basic values on any editor are `alias`, `name` and `editor`. These three **m
...
 
-* `view` Path to the HTML file to use for rendering the editor
+* `view` Path to the HTML file to use for rendering the editor.
 * `hideLabel` Turn the label on or off by using `true` or `false`, respectively.
-* `valueType` Sets the database type the value is stored as, by default it's `string`
-* `validation` Object describing required validators on the editor
-* `isReadOnly` Disables editing the value
+* `valueType` Sets the database type the value is stored as, by default it's `string`.
+* `validation` Object describing required validators on the editor.
+* `supportsReadOnly` Decide whether or not the property should support the read-only mode, which determines if the value of a shared/invariant property is greyed out or accessible.
```
Comment (Zeegaan) [inline Extending/Property-Editors/Package-Manifest/index.md:125]:
```suggestion
* `supportsReadOnly` Sets whether the editor supports read-only mode, if set to true, the editor is expected to have its own implementation of the read-only mode.
```

**Review by Zeegaan — CHANGES_REQUESTED** [review]
Found a small correction 👍

**Comment by sofietoft** [comment]
Thanks for helping getting this one right @Zeegaan ! 💪 

Seems we're good to go then, yeah? 😁

**Review by Zeegaan — APPROVED** [review]
@sofietoft yep, LGTM 👀

**Pull request merged**